app/models/concerns/offeror.rb

```diff
@@ -0,0 +1,99 @@
+module Offeror
+
```

Comment:
Perhaps GitHub represents it this way because originally indents were made by space.